README.md

```diff
@@ -138,30 +137,10 @@ datetime.date(2016, 4, 1)
 
 ```
 
-**Reading `.dbc` file:**
-
-```python
->>> from pysus.utilities.readdbc import read_dbc
```

Comment:
It's true that this function no longer exists within pysus, but for people that have standalone DBC files they can still use the `pyreaddbc.read_dbc` Perhaps this example could be modified to just fix the import, and be kept in the README

Reply:
But this function is not even there. There are open issues about it in the repo, too.

```
>>> from pyreaddbc import read_dbc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'read_dbc' from 'pyreaddbc' (/Users/ccc/Library/Caches/pypoetry/virtualenvs/pysus-Np31jkgz-py3.11/lib/python3.11/site-packages/pyreaddbc/__init__.py)
>>> from pyreaddbc import readdbc
>>> filename = '~/Downloads/DOAC2023.dbc'
>>> readdbc(filename, encoding='iso-8859-1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'module' object is not callable
```

Reply:
You are right. I am going to fix this, by readding this function to Pyreaddbc. I think it's useful